docs/en/operations/settings/query_complexity.md

```diff
@@ -207,22 +207,22 @@ What to do when the amount of data exceeds one of the limits: 'throw' or 'break'
 
 ## max_rows_in_join {#settings-max_rows_in_join}
 
-Limits number of rows in the hash table that is used when joining tables.
+Limits the number of rows in the hash table that is used when joining tables.
 
-This settings applies to [SELECT ... JOIN](../../query_language/select.md#select-join) operations and [Join engine](../table_engines/join.md) functioning.
+This settings applies to [SELECT ... JOIN](../../query_language/select.md#select-join) operations and the [Join table engine](../table_engines/join.md) functioning.
 
 ClickHouse can proceed with different actions when the limit is reached. Use the [join_overflow_mode](#settings-join_overflow_mode) settings to choose the action.
 
 Possible values:
 
 - Positive integer.
-- 0 — Memory control is disabled.
+- 0 — Rows control is disabled.
```

Comment:
"rows control" doesn't make sense

Reply:
What is make sense?

Reply:
for example "unlimited number of rows in join"